deploy-kyma-keda-k3d.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/bash
+
+# Add those to /etc/hosts
+# 127.0.0.1 k3d-kyma-registry.localhost
+# 127.0.0.1 k3d-kyma-registry
+
+# Provision K3D cluster
+kyma provision k3d --ci
+
+# Build image of keda manager and  OCI image of keda module
+make module-image IMG_REGISTRY=k3d-kyma-registry:5001/unsigned/operator-images
+make module-build IMG_REGISTRY=k3d-kyma-registry:5001/unsigned/operator-images MODULE_REGISTRY=k3d-kyma-registry.localhost:5001/unsigned
+
+#Adjust generated module template for single cluster mode on k3d
+cat template.yaml | yq '.spec.target = "control-plane"' - | sed -E "s/:5001/:5000/g" > template-k3d.yaml
+
+# Install lifecycle and module managers
+kyma alpha deploy --template=./template-k3d.yaml
```

Comment:
Based on experience, I would recommend using the `./bin/kyma-unstable` binary. We usually have the stable cli installed and the cli is updated too often with changes to this part of the code. It can cause incompatibility issues.